Gopkg.toml

```
@@ -19,7 +19,7 @@ ignored = ["google.golang.org/appengine*"]
   name = "golang.org/x/crypto"
 
 [[constraint]]
-  revision = "a646d33e2ee3172a661fc09bca23bb4889a41bc8"
+  revision = "4497e2df6f9e69048a54498c7affbbec3294ad47"
```

Can you use the constraint "master" ? The commit 4497e2df6f9e69048a54498c7affbbec3294ad47 is 3 week old. This is not blocking for the PR but why not updating to the latest.
```suggestion
   branch = "master"
```